CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+5. **Push and open a pull request:** push your branch and open a pull request with a clear description of your changes.
+
+## 🎉 From Code to PR: Your Guide to Submitting a Pull Request
+
+To ensure a smooth review process, please follow these guidelines when submitting a pull request:
+
+1. **Ensure your code is up-to-date:**
+   - Before starting your work, make sure to pull the latest changes from the `main` branch.
+   - If there are conflicts between your branch and `main`, resolve them locally before submitting your PR.
+
+2. **Branch naming:** use a descriptive branch name that reflects the changes.
+
+3. **Commit messages:**
+   - Write clear and concise commit messages that explain the purpose of the changes.
+   - Use the present tense, e.g., "Add login feature" instead of "Added login feature."
+
+4. **Description:**
+   - Provide a clear description of the changes made in the pull request.
+   - If the changes are related to a specific issue, reference in the PR description.
+
+5. **Ask questions:** ❓ if you’re not sure about something, don’t hesitate to ask! We’re here to help each other.
+
+6. **Include a demo:** please show the impact of your changes by including before/after screenshots or a short demo video. Visuals make everything better. 😎
+
+7. **Test your changes:** make sure to test your changes locally to ensure they work as expected.
+
+8. **Pull request title:**
+   - Use a descriptive title for your pull request, such as "Fix search button alignment".
+   - Follow [the conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0/)
+
+9. **Request a review:**
+   - Once your PR is ready, request a review from one of the maintainers.
+   - Be open to feedback and make changes as requested.
+
+## 🧐 Code Review Process
+
+### 🔍 How to Perform a Code Review
+
+If you are a contributor or maintainer and you want to review a pull request, please follow these guidelines:
+
+1. **Understand the changes:**
+   - Read the pull request description to understand the proposed changes.
+   - Look at the related issues (if mentioned) to understand the context.
+
+2. **Review the code:**
+   - Be a code detective! 🕵️‍♂️ Check the code for clarity, readability.
+   - Verify that the code is efficient and free of obvious bugs or security issues.
+
+3. **Provide constructive feedback:** 💬
+   - Leave comments on specific lines or sections of the code.
+   - Be constructive and polite in your feedback. Suggest improvements and explain why certain changes might be beneficial.
+   - Use phrases like "Consider...", "Could we...", or "It might be better to..." to keep the feedback positive.
+
+4. **Approve or request changes:**
+   - If the pull request is good to go, approve it.
+   - If in doubt, pull the branch locally and verify that it works as expected before approving it.
+   - If it needs changes, select "Request Changes" and leave detailed feedback on what should be improved.
+
+### 💌 How to Receive Code Review
```

Comment:
@Misofser, because this guide is first and foremost for the contributors, let's swap the "How to Receive Code Review" and "How to Perform a Code Review" changes. And then LGTM.